### DISCUSSION TITLE
Github Copilot not working for Visual Studio 2022

### DISCUSSION THREAD

**Discussion by reyniel-inriver** [community/community#102999] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I installed github copilot and copilot chat on my new machine, but it doesn't seem to work.
![image](https://github.com/community/community/assets/155035294/046a7ba6-b616-421d-8f98-f512f52d131b)

The version on my VS is 2022 Community Edition v17.8.6
![image](https://github.com/community/community/assets/155035294/72b31a75-7a84-4685-bc9d-329a663be4bf)

Here are the steps I have taken already.

> Restarted VS and my machine already a couple of times.

> Logged in to github using my account, with an active copilot subscription

![image](https://github.com/community/community/assets/155035294/602e1665-09dc-4b1b-be13-01bea619e977)

> Followed the steps on this link (https://github.com/orgs/community/discussions/50395), but there seems to be no copilot on my (Git -> Settings -> GitHub -> CoPilot)

![image](https://github.com/community/community/assets/155035294/3f21a62e-0a8d-49ec-b5cb-66fbb3065b24)

I don't know what else to do, since there isn't of documentations online about this kinds of issues.

**Comment by ayim** [comment]
Do you have both icons 
![image](https://github.com/community/community/assets/10951869/d7b092c4-25f3-454f-952c-ae2085f3738c)

and the Copilot Chat menu item here?
![image](https://github.com/community/community/assets/10951869/74e930a5-682c-4db2-b1bc-0c552c022513)

**Reply by reyniel-inriver** [reply]
No, those GitHub Copilot buttons are not available on my IDE up to now. @ayim

**Reply by ayim** [reply, marked as answer]
kk. Please file a ticket here (Report a Problem) so we can collect logs and take a closer look:.
We'll keep in contact via the VS Feedback tool once you file a ticket
![image](https://github.com/community/community/assets/10951869/cf4e53e4-93f3-4400-824f-b373e9d75de7)

**Comment by ayim** [comment]
@reyniel-inriver let us know - going to close this item in a day or two otherwise
